docker-compose.base.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.dev
-    command: python terraso_backend/manage.py runserver 0.0.0.0:${PORT}
+    command: daphne -b 0.0.0.0 -p ${PORT} "terraso_backend.config.asgi:application"
```

Comment:
question: on render, we have
>Optionally override your Dockerfile's CMD and ENTRYPOINT instructions with a different command to start your service.
```
gunicorn --pythonpath=terraso_backend/ config.wsgi -b 0.0.0.0:${PORT}
```

With this change, can we eliminate this override? Or do we change it to something else?